Packages/vcs/Lib/utils.py

```diff
@@ -1716,6 +1716,37 @@ def getworldcoordinates(gm, X, Y):
     return wc
 
 
+def rgba_color(color, colormap):
+    """Try all of the various syntaxes of colors and return 0-100 RGBA values."""
+    try:
+        # Is it a colormap index?
+        return colormap.index[color]
+    except ValueError:
+        # Is it a color tuple?
+        if len(color) == 3 or len(color) == 4:
+            for c in color:
+                try:
+                    int(c)
+                except:
+                    break
+            else:
```

Comment:
Yup, exceptions are cheap in python. [Ask for forgiveness, not permission](http://stackoverflow.com/a/12265860/725871)